pan_tilt:

 #http://answers.gazebosim.org/question/5468/gazebo_ros_pkgs-no-p-gain-specified-for-pid-in-positionjointinterface/
 gazebo_ros_control:
   pid_gains:
     pan_joint: {p: 100.0, i: 0.01, d: 10.0}
     tilt_joint: {p: 100.0, i: 0.01, d: 10.0}



